tests/lib/util/util.test.ts

```diff
@@ -288,6 +290,130 @@ describe('Utils', () => {
 		});
 	});
 
+	describe('getImage', () => {
+		test('GIVEN message w/ attachments w/ image w/o proxyURL attachment THEN returns url', async () => {
+			const filePath = resolve(__dirname, '..', '..', '..', 'tests', 'mocks', 'image.png');
+			const readStream = createReadStream(filePath);
+			const buffer = await utils.streamToBuffer(readStream);
```

Comment:
Any reason you don't directly fs.promises.readFile?

Reply:
It's... the point of testing ``streamToBuffer``...

Reply:
No...you're testing getImage, which has nothing to do with streamToBuffer?